internal/lib/container_server.go

```diff
@@ -24,7 +25,6 @@ import (
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	pb "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
-	"k8s.io/kubernetes/pkg/kubelet/dockershim/network/hostport"
```

Comment:
It would be better to have one commit switching over to the new internal hostport manager (and removing the unused files) and then a second commit adding dual-stack support. Among other things it may make it easier to revert back to the upstream hostport manager when/if that gets updated

Reply:
the code is going to be removed
https://github.com/kubernetes/kubernetes/pull/94624

Reply:
I mean, is not going to be a revert to the same folder